packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -4,12 +4,13 @@ import { PieIconButton } from '@justeattakeaway/pie-icon-button';
 import {
     WebComponentTestWrapper,
 } from '@justeattakeaway/pie-webc-testing/src/helpers/components/web-component-test-wrapper/WebComponentTestWrapper.ts';
-import { renderTestPieModal } from '../helpers/index.ts';
+import { createScrollablePageHTML, renderTestPieModal } from '../helpers/index.ts';
 
 import { PieModal } from '@/index';
 import { headingLevels } from '@/defs';
 
 const closeButtonSelector = '[data-test-id="modal-close-button"]';
+const modalSelector = '[data-test-id="pie-modal"]';
```

Comment:
Maybe we could use the class `componentSelector` here and also in the dialog attribute.

Reply:
I get changing the class to `componentSelector` but don't feel like using `componentSelector` for the attribute name is the right approach seeing as this will only be used for testing and having it called a `data-test-id` is more clear to people what the attribute is used for.